runtime/avn/src/lib.rs

```diff
@@ -224,7 +224,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("avn-parachain"),
     impl_name: create_runtime_str!("avn-parachain"),
     authoring_version: 1,
-    spec_version: 52,
+    spec_version: 37,
```

Comment:
how does this change from a number in the 50s to 37?

Reply:
this was due to experimenting on the local change